README.md

```diff
@@ -9,52 +9,3 @@ The primary goal is to allow high level contributions by users that maybe intere
 
 We reuse the knowledge panel format, which remove the need for the application to change as the server side code change, or to implement specific business logic.
 
```

Comment:
you should put a link to the sub documentations:
- project setup
- testing
- api references

Reply:
done!